docs/package.json

```diff
@@ -8,13 +8,28 @@
     "build": "yarn build:api && yarn build:next",
     "start": "next start",
     "build:next": "next build",
-    "build:api": "api-documenter markdown --input-folder . --output-folder src/pages/reference/"
+    "build:api": "yarn ts-node --compiler-options '{\"module\":\"commonjs\"}' tools/apiExtractorUtils.ts"
```

Comment:
Do I understand correctly, now we're having a custom server that will extract the articles & the source of this server is the `tools/apiExtractorUtils.ts` file?

Reply:
It's not a server, it's a script.